src/plugins/vis_type_timeline/config.ts

```diff
@@ -35,7 +35,9 @@ export const configSchema = schema.object(
     enabled: schema.boolean({ defaultValue: true }),
     ui: schema.object({ enabled: schema.boolean({ defaultValue: false }) }),
     graphiteAllowedUrls: schema.maybe(schema.arrayOf(schema.string())),
+    /** @deprecated use graphiteDeniedIPs*/
     graphiteBlockedIPs: schema.maybe(schema.arrayOf(schema.string())),
+    graphiteDeniedIPs: schema.maybe(schema.arrayOf(schema.string())),
```

Comment:
same as the above one we should use this instead of creating two: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1bb7a4f0d740879befc7aadc0b47e87cc67500cb/src/plugins/vis_type_timeline/server/index.ts#L42

Reply:
we should use renameFromRootWithoutMap, that way the code doesn't change and the end users will get this information when running as labeling it with the annotation will help the user. As you can from the above file https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1bb7a4f0d740879befc7aadc0b47e87cc67500cb/src/plugins/vis_type_timeline/server/index.ts#L42

you can add `renameFromRootWithoutMap('graphiteBlockedIPs', 'graphiteDeniedIPs')`, then we don't have to add any extra logic or code.

Reply:
fixed!